packages/runtime/test/lib/testUtils.ts

```diff
@@ -700,12 +702,12 @@ export async function waitForRecipientToReceiveNotification(
 
 /**
  * The owner of a RelationshipAttribute receives a Request of a
- * peer and forwards them the ThirdPartyRelationshipAttribute,
+ * peer and forwards them the RelationshipAttribute,
```

Comment:
I'd strongly argue for keeping the term ThirdPartyRelationshipAttribute here for clarity. If you are unhappy with this specific wording, how about "... and sends them a ThirdPartyRelationshipAttribute"?

Reply:
As @sebbi08 correctly pointed out, this function can also be used to request existing RepositoryAttributes. In case of ThirdPartyRelationshipAttributes I just noticed that the Identity receiving the Request doesn't necessarily have to be the owner of the source RelationshipAttribute. Hence, to avoid confusion I suggest to rename `owner` to `responder` and `peer` to `requestor`. For clarity I'd change the function's description to: "The requestor asks the responder for a RepositoryAttribute or a RelationshipAttribute from another Relationship. The responder sends them an own shared IdentityAttribute or ThirdPartyRelationshipAttribute, waiting for all communication and event processing to finish. Returns the responder's own shared Attribute."